python/poetry.lock

```diff
@@ -1,4 +1,4 @@
-# This file is automatically @generated by Poetry 2.0.1 and should not be changed by hand.
+# This file is automatically @generated by Poetry 2.1.1 and should not be changed by hand.
```

Comment:
This doesn't make sense unless the Poetry version is also updated in pyproject.toml